src/graph/graphcontext.c

```diff
@@ -122,6 +128,14 @@ void GraphContext_MarkWriter(RedisModuleCtx *ctx, GraphContext *gc) {
 	RedisModule_FreeString(ctx, graphID);
 }
 
+inline void GraphContext_MarkInDecode(GraphContext *gc) {
+	GraphDecodeContext_SetGraphInDecode(gc->decoding_context);
+}
+
+inline bool GraphContext_IsInDecode(GraphContext *gc) {
+	return GraphDecodeContext_IsGraphInDecode(gc->decoding_context);
+}
```

Comment:
I don't really feel like this level of indirection gives much benefit. Thoughts?